### PR TITLE
Update orbitControls.js

### PR DESCRIPTION
**Pull request by hrgdavor**
### All Submissions:
* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?
* [x] Does your submission pass tests?

a small fix that modifies pan so it is scaled based on current zoom (distance camera<->eye).

There is a comment near my change that somebody left there
```
  // TODO scale by the correct near/far value instead of 1000 ?
  // const planesDiff = camera.far - camera.near
```

but when I inspected near and far, it was the same regardless of zoom. Maybe this is related to some other aspect.

It is a simple improvement, please test if you find problems with the change.

### CODE REVIEW

**Comment by hrgdavor** [comment]
Also works better on mobile, and is more aligned with fingers than current jscad.xyz

File: packages/utils/regl-renderer/src/controls/orbitControls.js [hunk 1]
```diff
@@ -250,7 +250,8 @@ const pan = ({ controls, camera, speed = 1 }, delta) => {
   const unPanEnd = unproject([], panEnd, viewport, invProjView)
   // TODO scale by the correct near/far value instead of 1000 ?
```
Comment (z3dev) [inline packages/utils/regl-renderer/src/controls/orbitControls.js:251]: I think this fix addresses this TODO, so let's remove the comments.

**Review by z3dev — CHANGES_REQUESTED** [review]
Just a small change requested to remove comments.

I tried this on all browsers on MacOS and iOS. Works really nice. SUPER JOB!

**Review by z3dev — APPROVED** [review]
Looks nice!
Thanks for the late change.

**Pull request merged**